N/NetCDF/common.jl

```diff
@@ -89,12 +112,20 @@ nc-config --all
             Dependency("nghttp2_jll", v"1.40.0"),
         ],
         v"1.6" => [
-            Dependency("LibCURL_jll"),
+            Dependency("LibCURL_jll", v"7.73.0"),
             # The following libraries are dependencies of LibCURL_jll which is now a
             # stdlib, but the stdlib doesn't explicitly list its dependencies
-            Dependency("LibSSH2_jll"),
+            Dependency("LibSSH2_jll", v"1.9.1"),
             Dependency("MbedTLS_jll", v"2.24.0"),
             Dependency("nghttp2_jll"),
+        ],
+        v"1.8" => [
+            Dependency("LibCURL_jll", v"7.81.0"),
+            # The following libraries are dependencies of LibCURL_jll which is now a
+            # stdlib, but the stdlib doesn't explicitly list its dependencies
+            Dependency("LibSSH2_jll", v"1.10.2"),
+            Dependency("MbedTLS_jll", v"2.28.0"),
+            Dependency("nghttp2_jll"),
```

Comment:
The comment hasn't been true for quite some time (and it has been true for only few days).  Did you try removing these lines?

Reply:
Indeed, the compilation works if I remove the indirect dependencies. The PR is updated.